common/src/main/java/com/wynntils/models/gearinfo/tooltip/GearTooltipFooter.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gearinfo.tooltip;
+
+import com.wynntils.models.concepts.Powder;
+import com.wynntils.models.gearinfo.type.GearInfo;
+import com.wynntils.models.gearinfo.type.GearInstance;
+import com.wynntils.models.gearinfo.type.GearMajorId;
+import com.wynntils.models.gearinfo.type.GearRestrictions;
+import com.wynntils.models.gearinfo.type.GearTier;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.utils.mc.RenderedStringUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+
+public final class GearTooltipFooter {
+    private static final int PIXEL_WIDTH = 150;
+
+    public static List<Component> buildTooltip(GearInfo gearInfo, GearInstance gearInstance) {
+        List<Component> footer = new ArrayList<>();
+
+        // major ids
+        // FIXME: Missing "<+Entropy: >Meteor falls..." which should be in AQUA.
```

Comment:
Is Entropy formatted differently by the game?

Reply:
Upon further inspection of the code, it seems to me that _every_ major ID would be missing its aqua-colored name, unless I'm missing something

Reply:
Yeah, sorry, the comment was misleading, it should read "for example, like this:"